src/core/components/courses/TraineeTable.vue

```diff
@@ -28,8 +28,8 @@
     </div>
 
     <!-- Add trainee modal -->
-    <trainee-creation-modal v-model="traineeCreationModal" :new-trainee="newTrainee" :company-options="companyOptions"
-      :first-step="firstStep" :add-new-trainee-company-step="addNewTraineeCompanyStep" :is-intra-course="isIntraCourse"
+    <learner-creation-modal v-model="traineeCreationModal" :new-user="newTrainee" :company-options="companyOptions"
+      :first-step="firstStep" :identity-step="addNewTraineeCompanyStep" :company-step="isIntraCourse"
```

Comment:
en terme de sens ici, quand `companyStep` est `true` tu veux afficher le champ select sur la structure dans la modal 
donc la props c'est `:company-step="!isIntraCourse"`
et dans la modal il faut changer `<ni-select v-if="companyStep" in-modal v-model.trim="newUser.company"` sinon c'est bizarre